src/index.js

```diff
@@ -52,8 +52,10 @@ export { ProductTileContent } from './Tile/Tile';
 export { ProductTileMedia } from './Tile/Tile';
 export { SearchInput } from './SearchInput/SearchInput';
 export { SideNav, SideNavList, SideNavListItem } from './SideNavigation/SideNavigation';
+export { Tab } from './Tabs/Tab';
+export { TabContent } from './Tabs/TabContent';
```

Comment:
I think the `TabContent` component should be internal-only and should not be exported.